frontend/views/containers/TimeTravel.vue

```diff
@@ -66,12 +64,12 @@ export default {
       },
       config: {
         sliderConfig: {
-          tooltip: 'focus',
           marks: true,
           adsorb: true,
           lazy: true,
           min: 0,
-          max: 0
+          max: 1,
```

Comment:
This is why you're getting the exception, it's because it will result in an index out of bounds.